src/core_modules/capture-core/components/Pages/Enrollment/hooks.js

```diff
@@ -11,3 +13,42 @@ export const useEnrollmentInfo = (enrollmentId: string, programId: string) => {
 
     return { programHasEnrollments, enrollmentsOnProgramContainEnrollmentId, tetId };
 };
+
+
+const extractDataElements = (data) => {
+    if (!data?.programData) { return {}; }
+    return data.programData.programStages
+        .reduce((acc, curr) => {
+            curr.programStageDataElements.forEach((stage) => {
+                acc[stage.dataElement.id] = { ...stage.dataElement };
+            });
+            return acc;
+        }, {});
+};
+
+// $FlowFixMe
+export const useRunRulesForEnrollement = (orgUnit, program, programStagesData, teiAttributes) => {
```

Comment:
Hey,
Missing types for `orgUnit, program, programStagesData, teiAttributes` parameters
Thank you!